### ISSUE TITLE
Timeseries data example

### ISSUE THREAD

**Issue by Samvid95** [adap/flower#758]
Is there any Flower example available for the time series models?

**Comment by Samvid95** [comment]
Hey everyone, is there any update on this? Is there any example available?

**Comment by danieljanes** [comment]
Thanks for reaching out @Samvid95 @prathapkumarbaratam . Did you make progress on this, and if so, would you be interested in contributing a new Flower example?

**Comment by Samvid95** [comment]
Yeah definitely.  I will take a stab on this and create a quick start example.

**Comment by prathapkumarbaratam** [comment]
@Samvid95  hi samvid, Iam also working on that time series example. it would be great if we work together. connect me here : https://www.linkedin.com/in/baratam-prathap-kumar/

**Comment by WilliamLindskog** [comment]
Hi @Samvid95, @prathapkumarbaratam 

Thanks for raising this. Is this an example that you'd like to contribute with? Happy to review a PR if you wish to open this.

**Comment by WilliamLindskog** [comment]
Good feature request, migrating this to [Flower Discuss](https://discuss.flower.ai/t/add-time-series-data-example-to-flower/641). Please feel free to read more there and open PR if there is interest.

**Comment by WilliamLindskog** [comment]
Hi @prathapkumarbaratam, did you complete an example for time-series based FL? Would you want to merge it to Flower if you have time?

**Comment by WilliamLindskog** [comment]
Hi @Samvid95, @prathapkumarbaratam, 

would you be interested in contributing with such example? Happy to help out during PR process.

**Comment by WilliamLindskog** [comment]
implemented a federated example using time-series and feature extraction yesterday, might come to merge soon.
